docker/docker-compose.2204.57.yaml

```diff
@@ -12,7 +12,8 @@ services:
   test:
     image: swift-certificates:22.04-5.7
     environment:
-      - MAX_ALLOCS_ALLOWED_parse_webpki_roots=200050
+      - SWIFT_VERSION=5.7
+      - MAX_ALLOCS_ALLOWED_parse_webpki_roots=422050
       - MAX_ALLOCS_ALLOWED_tiny_array_cow_append_contents_of=9050
```

Comment:
Any reason not to drop the `MAX_ALLOCS_ALLOWED` variables?

Reply:
No, just an oversight. removed them now.